biztalk/core/azure-blob-adapter.md

```diff
@@ -0,0 +1,153 @@
+    | **Blob name** | Name of the blob used by adapter to upload the message to Blob storage container. Macros can be used for Blob name similar to file adapter. For available Macros please refer to [Using macros in file names](restrictions-when-configuring-the-file-adapter.md#using-macros-in-file-names). |
+    | **Namespace for blob metadata** | Context properties of the message can be written to blob metadata if the namespace of the property matches this field. |
+
+    When finished, your properties look similar to the following: 
+
+    ![General properties](../core/media/BlobAdapter-send-General.png)
+
+5. Configure the **Advanced** properties:
+
+    |Use this|To do this|  
+    |---|---|  
+    |**Blob type**| Azure Blob adapter can upload message as different blob types as needed, visit [blob types](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction) for more information. |
+    |**Write mode**| In case of same blob name already exists, different **Write mode** are provided. |  
+
+6. Select **Ok** to save your changes. 
+
+
+### Test your send port
+
+You can use a simple File receive port and location to send messages to your Azure Blob storage. 
+
+1. Create a receive port named "FileReceivePort" using the File adapter. Within your receive location,  set the **Receive folder** to **C:\Temp\In\\**, and set the file mask to **\*.xml**.
+2. In your Azure Blob storage send port properties, set the **Filters** to `BTS.ReceivePortName == FileReceivePort`.
+3. Paste the following into a text editor, and save the file as **AzureBlobStorageMessage.xml**. This is your sample message. 
+
+    ```xml
+    <Data>
+      <DataID>DataID_0</DataID>
+      <DataDetails>DataDetails_0</DataDetails>
+    </Data>
+    ```
+
+4. Start the File receive location and the Azure Blob storage send port.
+5. Copy **AzureBlobStorageMessage.xml** sample message into the receive folder (C:\Temp\In\). The send port sends the XML file to the Azure Blob storage. You can verify that by looking into your Azure storage container and exam the new created or updated file.
+
+## Receive messages from Azure Blob storage
+
+1. In the BizTalk Server Administration console, right-click **Receive Ports**, select **New**, and select **One-Way receive port**. 
+
+    [Create a receive port](../core/how-to-create-a-receive-port.md) provides some guidance.
+
+2. Enter a name, and select **Receive Locations**. 
+
+3. Select **New**, and **Name** the receive location. In **Transport**, select **AzureBlobStorage** from the **Type** drop-down list, and then select **Configure**. 
+
+4. Configure the **Azure Account** properties: 
+
+    |Use this|To do this|  
+    |---|---|  
+    | **Sign-in** | Sign into your Azure account |
+    | **Subscription** | Select the subscription that has your Azure storage account |
+    | **Resource Group** | Select your resource group that has your Azure storage account |
+
+4. Configure the **Endpoint** properties: 
+
```

Comment:
Something does not seem right here related to Even hub?

Reply:
Good catch. Seems i messed up here when we were exploring the correct way to work with github. Corrected.